language/use-cases/sql-code-generation/sql_code_generation_langchain.ipynb

```diff
@@ -0,0 +1,1186 @@
+{
```

Comment:
<p>This seems redundant. Shouldn't this be a one-time setup when defining the <code>generate_prompt</code> function? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/404/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
<p>removed.</p>